tests/core/mocksettings.py

```diff
@@ -0,0 +1,6 @@
+foo = "bar"
+
+d = {
+    "k1": "v1",
+    "k2":{"a":"a","b":"b"}
+}
```

Comment:
This mockup does not need its own file, but could be provided as a property or created in `setUp` in the test suite